projects/aca-content/src/lib/ui/theme.scss

```diff
@@ -14,6 +14,12 @@ mat-icon {
   vertical-align: middle;
 }
 
+button {
+  #{$mat-icon} {
+    margin-top: -10px;
```

Comment:
this will break the layout for the icon buttons that previously were aligned properly, for example in toolbar